pkg/operator/operator.go

```diff
@@ -175,7 +176,7 @@ func withUserAgent(sess *session.Session) *session.Session {
 func checkEC2Connectivity(ctx context.Context, api *ec2.EC2) error {
```

Comment:
Do we need this function check in the beginning now if the liveness/readiness is doing this now?